services/authnz/values-int.yaml

```diff
@@ -0,0 +1,86 @@
+secrets:
```

Comment:
Since the authnz chart is technically a subchart (with the name authnz that you gave it in the requirements.yaml), you'll want to take this whole file, and put it under an authnz block, like:

```
authnz:
    ... (all the stuff)
```

Otherwise it will get confused and not read your values files

Reply:
Same for stable